docs/admin/terraform/README.md

```diff
@@ -0,0 +1,118 @@
+---
+tags:
+  - administration
+  - terraform
+  - operations
+  - automation
+  - IaC
+  - Infrastructure as Code
+---
+
+## Benefits of Terraform and IaC
+
+**Consistency:** you create one Terraform configuration and use it to configure environments repeatedly.
+
+**Minimized human error:** the configuration is now `as code` and is deployed automatically.
+
+**Fast:** with a single command, you can have all your configuration provisioned.
+
+**Observability:** the configuration are made `as code`, you now have one place to review all the configuration and who changed what.
+
+
+## Janssen Terraform provider
+
+!!! Note
+    The Janssen Terraform resources cannot be created or destroyed/deleted using Terraform, as they are always present in a Janssen deployment. Instead, they can be imported and then updated. The creation of such a resource using `terraform apply` will result in an error. Deletion on the other hand using `terraform destroy` will result in the resource being removed from the state file and become not under Terraform management.
+
+The [Janssen](https://registry.terraform.io/providers/JanssenProject/jans/latest/docs) Terraform provider is used to manage resources in a Janssen deployment. This includes all configurations, users, groups, OIDC clients, and more.
+
+## Example - Configure Janssen
+
+Let's have an example on `importing` the current `logging level` of a deployment and `changing` it using Terraform.
+
+1. Configure Terraform to install the required plugins for `Janssen provider`. Add this to your `.tf` file:
```

Comment:
For the example , show the directory tree with each of the .tf files. 
- backend.tf - use local for an example and advise to use buckets
- clients.tf - If using this in examples this should hold the clients
- configurations.tf - This holds the instance configurations
- scopes.tf - This holds the scopes when used
- scripts.tf - This holds the scripts when used
- variables.tf
- versions.tf